diff --git a/src/index.ts b/src/index.ts
index 5e099a50ca782ada322e4d654ad0fac27bf3e967..5d36c7c364a23d99e0e5cdd1c6a71446e953652b 100644
--- a/src/index.ts
+++ b/src/index.ts
@@ -4,7 +4,7 @@ import type { AdbCredentialStore, AdbPrivateKey } from "@yume-chan/adb";
 
 function openDatabase() {
     return new Promise<IDBDatabase>((resolve, reject) => {
-        const request = indexedDB.open("Tango", 1);
+        const request = indexedDB.open("nyaadenwa", 1);
         request.onerror = () => {
             reject(request.error!);
         };
